packages/babel-plugin-transform-runtime/src/runtime-corejs3-definitions.js

```diff
@@ -185,47 +185,51 @@ export default () => {
     // See ./helpers.js@typeAnnotationToString for the supported types
 
     InstanceProperties: {
-      at: { stable: false, path: "at" },
-      bind: { stable: true, path: "bind" },
-      codePointAt: { stable: true, path: "code-point-at" },
-      codePoints: { stable: false, path: "code-points" },
+      at: { stable: false, path: "at", types: ["string"] },
+      bind: { stable: true, path: "bind", types: ["function"] },
+      codePointAt: { stable: true, path: "code-point-at", types: ["string"] },
+      codePoints: { stable: false, path: "code-points", types: ["string"] },
       concat: { stable: true, path: "concat", types: ["array"] },
-      copyWithin: { stable: true, path: "copy-within" },
-      endsWith: { stable: true, path: "ends-with" },
-      entries: { stable: true, path: "entries" },
-      every: { stable: true, path: "every" },
-      fill: { stable: true, path: "fill" },
-      filter: { stable: true, path: "filter" },
-      find: { stable: true, path: "find" },
-      findIndex: { stable: true, path: "find-index" },
-      flags: { stable: true, path: "flags" },
-      flatMap: { stable: true, path: "flat-map" },
-      flat: { stable: true, path: "flat" },
-      forEach: { stable: true, path: "for-each" },
-      includes: { stable: true, path: "includes" },
-      indexOf: { stable: true, path: "index-of" },
-      keys: { stable: true, path: "keys" },
-      lastIndexOf: { stable: true, path: "last-index-of" },
-      map: { stable: true, path: "map" },
-      matchAll: { stable: false, path: "match-all" },
-      padEnd: { stable: true, path: "pad-end" },
-      padStart: { stable: true, path: "pad-start" },
-      reduce: { stable: true, path: "reduce" },
-      reduceRight: { stable: true, path: "reduce-right" },
-      repeat: { stable: true, path: "repeat" },
-      replaceAll: { stable: false, path: "replace-all" },
-      reverse: { stable: true, path: "reverse" },
-      slice: { stable: true, path: "slice" },
-      some: { stable: true, path: "some" },
-      sort: { stable: true, path: "sort" },
-      splice: { stable: true, path: "splice" },
-      startsWith: { stable: true, path: "starts-with" },
-      trim: { stable: true, path: "trim" },
-      trimEnd: { stable: true, path: "trim-end" },
-      trimLeft: { stable: true, path: "trim-left" },
-      trimRight: { stable: true, path: "trim-right" },
-      trimStart: { stable: true, path: "trim-start" },
-      values: { stable: true, path: "values" },
+      copyWithin: { stable: true, path: "copy-within", types: ["array"] },
+      endsWith: { stable: true, path: "ends-with", types: ["string"] },
+      entries: { stable: true, path: "entries", types: ["array"] },
```

Comment:
Does this work? (also for keys/values)

```js
Object.entries;
```

Reply:
> Does this work?

Yes, it's treated as unknown.

Reply:
Yes, that's a static method rather than an instance method.